README.md

```diff
@@ -8,6 +8,7 @@
 [![Maven Central](https://img.shields.io/maven-central/v/com.github.sonus21/rqueue-core)](https://repo1.maven.org/maven2/com/github/sonus21/rqueue-core)
 [![Javadoc](https://javadoc.io/badge2/com.github.sonus21/rqueue-core/javadoc.svg)](https://javadoc.io/doc/com.github.sonus21/rqueue-core)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+[![Project Dashboard](https://sourcespy.com/shield.svg)](https://sourcespy.com/github/sonus21rqueue/)
```

Comment:
Should it be called the source code dashboard? Github also has project dashboard

Reply:
You have a point. Updated.